src/coreclr/tools/Common/Compiler/CompilationBuilder.cs

```diff
@@ -25,6 +25,7 @@ public abstract partial class CompilationBuilder
         protected OptimizationMode _optimizationMode = OptimizationMode.None;
         protected int _parallelism = -1;
         protected bool _resilient;
+        protected ulong _imageBase;
```

Comment:
Could you move this to src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilationBuilder.cs?

This setting only makes sense for ReadyToRun and CompilationBuilder.cs is shared with the NativeAOT compiler.

Reply:
Yes, thanks.